phpunit.xml.dist

```diff
@@ -1,12 +1,12 @@
 <phpunit
```

Comment:
I was having a lot of trouble getting PHPUnit to find the config in the tests/config file and it was going to take longer than reasonable to figure out why it wouldn't listen to its own -c=tests/config/phpunit.xml.dist on scripts/test.sh.

Should I still try to suss it out before merging?